### DISCUSSION TITLE
time series data returned from WPS v7 to v8

### DISCUSSION THREAD

**Discussion by uwer** [TerriaJS/terriajs#6603] [General]
Hi guys, 
under v7 we had a WPS return complex data in form of a CSV that was displayed as time series at the bottom map view, this is now broken in V8 and there are no details in the migration guide. The data is returned  as 
<wps:ComplexData mimeType="application/vnd.terriajs.catalog-member+json" encoding="" schema="https://tools.ietf.org/html/rfc7159"><![CDATA[{"data": "date,ModelElevation\n2017-02-24T15:00:00+0000,-0.013 ..... \n2017-04-24T00:00:00+0000,-0.174", "type": "csv", "isEnabled": "true", "name": "ModelData", "tableStyle": {"columns": {"ModelElevation": {"units": "[m]", "active": "true"}}}}]]></wps:ComplexData>

Could anybody shed light on if this is still a feasible way of defining time series and if so what needs to change to make this work?
Cheers
UR

**Comment by nf-s** [comment]
Hi @uwer 

Can you please provide the Terria JSON in the `ComplexData`?
The TerriaJS JSON schema has changed quite a lot form version 7 to 8.

There are two options for WPS with version 7 TerriaJS JSON:

- You can upgrade to version 8 using our catalog converter - https://catalog-converter.terria.io/
- You can add `"forceConvertResultsToV8": true` to the WPS catalog function in your Terria Catalog

Cheers,
Nick

**Reply by uwer** [reply]
Thanks Nick,
this is actually a request I put out as it is part of the NationalMap-Aremi services transition to V8
so - i run the WPS service, but have no control over the static setup of NationalMap ingesting the services (even though they are defined through CSW)
hhhhmm - I did copy the content of ComplexData - but it must have stripped it in the message
you can see the failed request here 
https://nationalmap.gov.au/#share=s-9TZPhSaW8WZbW0oGMHFBX8icjEc
